### ISSUE TITLE
resource/image file added from VS UI will not be built into PRI file for WPF project

### ISSUE THREAD

**Issue by huichen123** [microsoft/WindowsAppSDK#1674]
**Describe the bug**
In a WPF project, adding resource image file from VS UI, the project is automatically modified to exclude those files. Thus they are not built into final PRI file.

**Steps to reproduce the bug**
1. Create a new WPF project
2. Reference to WindowsAppSDK 1.0-preview3
3. add a new resources.resw file or an existing image file to the project. The csproj file automatically adds something like:
~~~
  <ItemGroup>
    <Content Remove="SplashScreen.scale-200.png" />
  </ItemGroup>

  <ItemGroup>
    <PRIResource Remove="Resources.resw" />
  </ItemGroup>
~~~
4. build
5. The image and resource strings are not built into PRI file

**Expected behavior**
The image and resource strings are built into PRI file

**Screenshots**

**Version Info**

NuGet package version: 
[Microsoft.WindowsAppSDK 1.0.0-preview3] 

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |

**Additional context**
This seems to be a regression caused by https://github.com/microsoft/WindowsAppSDK/pull/1389. Workaround is to manually remove those ItemGroup added automatically.

**Comment by soIsWinUI3PackorUnpack** [comment]
> The image and resource strings are not built into PRI file

**so is WinUI 3** (both Packaged and Unpackaged are affected --> xbfs files are not built into pri file). 
see https://github.com/microsoft/WindowsAppSDK/discussions/1248#discussioncomment-2195280

**Comment by rohanp-msft** [comment]
This issue cannot be properly addressed in the Windows App SDK. It requires .NET SDK support. Will be filing feature request.

**Comment by rohanp-msft** [comment]
> > The image and resource strings are not built into PRI file
> 
> **so is WinUI 3** (both Packaged and Unpackaged are affected --> xbfs files are not built into pri file). see [#1248 (comment)](https://github.com/microsoft/WindowsAppSDK/discussions/1248#discussioncomment-2195280)

Hi @ghost, the two issues are unrelated. In fact, with the referenced issue, the XBFs _are_ embedded in the PRI, it's just that they're also loosely deployed (that's for WAPPROJ based apps; single-project MSIX Packaging doesn't have that issue).

**Comment by rohanp-msft** [comment]
This issue is fixed by the merged PR above if .NET SDK v6.0.300 or higher is used. It will be available in 1.1 Stable.
